learningequality/studio
# Basic integration of updated kolibri-tools

## Summary
### Description of the change(s) you made
* Updates kolibri-tools and adds fixes to webpack build process to handle the updated dependencies
* Updates language info to limit it to Studio supported languages
* Updates makefile commands for i18n to be Studio specific, remove copypasta from kolibri

### Manual verification steps performed
Ran `make i18n-extract`
Ran `yarn run devserver-hot`
Ran `yarn run build`

All seemed to do what I expected.
